### ISSUE TITLE
cursor is invisible but still working

### ISSUE THREAD

**Issue by 4rtemis-4rrow** [WillPower3309/swayfx#317]
- swayfx version 0.4

- **Debug Log:**
https://gist.github.com/4rtemis-4rrow/cad3fafc6ab89d4683764874a2d385e2

- **Configuration File:**
github.com/4rtemis-4rrow/dotfiles
note that the issue is not with my configs, since I hadn't changed my sway configs in a long time, infact, the issue appeared after my computer froze and I had to hard reboot it

- **Description:**
  mouse is still working, wev is still showing it's movements, and buttons are still highlighted when I hover over them, the cursor itself it invisible though, I tried changing my cursor theme, restarting sway, restarting my whole pc, deleting ~/.cache, none of that fixed it, I posted the issue to reddit, and no one responded, so I am kinda stuck

if you took the time to read this, thank you

**Comment by PaddiM8** [comment]
I'm having the same problem. On nvidia if that matters

**Comment by PaddiM8** [comment]
I wouldn't consider that a real solution though, since hardware cursors are better. Should probably keep the issue open

**Comment by 4rtemis-4rrow** [comment]
yeah I'd have to agree, hardware cursors are better, but considering that the common denominator is nvidia GPUs, that'd be (to say the least) annoying to fix, so until we have better nvidia drivers
